grpcutil/healthcheck/health_check.go

```diff
@@ -0,0 +1,53 @@
+package healthcheck
```

Comment:
[nit] Do we need this subpackage? I'm wondering if moving it to `grpcutil` is enough.

Reply:
@pracucci let me look into it.